app/ui/legacy/src/main/java/com/fsck/k9/activity/setup/AccountSetupAccountType.kt

```diff
@@ -27,6 +30,9 @@ class AccountSetupAccountType : K9Activity() {
 
     private lateinit var account: Account
     private var makeDefault = false
+    private lateinit var defaultAccountSettings: DefaultAccountSettings
+
+    private val backendManager = get(BackendManager::class.java)
```

Comment:
`DI.get()` is a helper that is only required for Java classes. In Kotlin you can write this as

```kotlin
private val backendManager: BackendManager by inject()
```

Reply:
Good spotting, I've made that change now. Thanks